content/intro-to-storybook/react/en/simple-component.md

```diff
@@ -115,16 +111,13 @@ When creating a story we use a base `task` arg to build out the shape of the tas
 
 ## Config
 
-We'll need to make a couple of changes to the Storybook configuration so it notices not only our recently created stories, but also allows us to use the CSS file that was introduced in the [previous chapter](/intro-to-storybook/react/en/get-started).
+We'll need to make a couple of changes to Storybook's configuration files so it notices not only our recently created stories, but also allow us to use the application's CSS file (located in `src/index.css`).
```

Comment:
```suggestion
We'll need to make a couple of changes to Storybook's configuration files so it notices our recently created stories and allows us to use the application's CSS file (located in `src/index.css`).
```

Reply:
Fixed with the latest commit. Marking this as resolved